Textile/Classes/RequestsHandler.m

```diff
@@ -45,8 +45,10 @@ - (instancetype)init {
 - (void)flush {
   // We don't know what thread we're being called on here, so dispatch to our
   // serial queue to make sure only one call to flush can be processed at a time
+  [self.node flushLock];
```

Comment:
I've exposed a lock that needs to be acquired here in order to ensure that a pending stop request is not honored until this async job is complete.